.github/workflows/test-enable-alma9-agent.yml

```diff
@@ -0,0 +1,28 @@
+name: Test the enabling of the Wazuh agent - AlmaLinux 9 - RPM
```

Comment:
Could we define an array of RPM manager-supported operating systems and loop into it? And another one for the agent? Could you please add all manager-supported RPM operating systems?
In addition, it could be great if we test Debian too.